feature-group-definitions/device-orientation-events.yml

```diff
@@ -0,0 +1,16 @@
+name: Device orientation events (intial support)
```

Comment:
Can you add a TODO here about the reference frame so that we can come back to it?

Reply:
Done!